Header header
bool st_arduino_connection
uint32 st_arduino_age
int32 st_arduino_sync_offset
uint32 st_arduino_ip_addr
uint8 st_arduino_imu_rate
uint16 st_arduino_imu_id
float32 st_arduino_imu_temp
uint32 st_arduino_ptp_interval